packaging/docker/samples/python/docker-compose.yml

```diff
@@ -21,21 +21,29 @@ version: '3'
 services:
   # Specify three fdbserver processes.
   fdb-coordinator:
-    image: foundationdb:6.1.8
+    image: foundationdb/foundationdb:6.2.11
```

Comment:
This needs to be bumped up for each release manually. Can we use some variable in YAML file so that it has to be updated once?

Also we can potentially use `./build/get_version.sh versions.target` to get the latest version. However, the `fdb.set_version()` still will stay old, so not sure if this would be super useful.